core/src/main/java/nu/yona/server/rest/TestController.java

```diff
@@ -52,6 +60,11 @@
 	@Autowired
 	private FirebaseService firebaseService;
 
+	@Autowired
+	private PassThroughHeadersHolder headersHolder;
+
+	private final CyclicBarrier barrier = new CyclicBarrier(2);
```

Comment:
For clarity `passThroughHeadersRequestBarrier` is more speaking here

Reply:
Agreed.